frontend/components/transaction/BidTransaction.vue

```diff
@@ -140,17 +118,10 @@ export default Vue.extend({
             type: Boolean,
             default: true,
         },
-        transactionAmountDai: {
-            type: Object as Vue.PropType<BigNumber>,
-            default: null,
-        },
-        minimumBidDai: {
-            type: Object as Vue.PropType<BigNumber>,
-            default: null,
-        },
     },
     data() {
         return {
+            transactionAmountDai: undefined as BigNumber | undefined,
```

Comment:
We store the same value under 3 different names:
- `amountToBid`
- `transactionAmountDai`
- `inputBidAmount` (although this also can contain `undefined`, so it's technically different)

Would it make sense to refactor:
- `amountToBid` -> `transactionBidAmount`
- `transactionAmountDai` -> `transactionBidAmount`

therefore leaving just two `inputBidAmount` and `transactionBidAmount`?